### ISSUE TITLE
The specific parameters of training

### ISSUE THREAD

**Issue by wwn1233** [pfnet-research/FSCS#4]
Wonderful work.  I want to know the specific training parameters, such as learning rate, epochs, m_loss_lambda and so on !
